src/Microsoft.PowerShell.Commands.Management/commands/management/GetComputerInfoCommand.cs

```diff
@@ -558,13 +558,13 @@ private static MiscInfoGroup GetOtherInfo(CimSession session)
             // get secure-boot info
             //TODO: Local machine only? Check for that?
             FirmwareType fwType = FirmwareType.Unknown;
-            if (Native.GetFirmwareType(ref fwType))
+            if (GetFirmwareType(ref fwType))
```

Comment:
The idea behind the wrapper methods is to have the property in the cmdlet's output object remain null if the native function fails or if the PInvoke fails to load or find the native function. The properties on that object that are integers, enums, etc., are all Nullable<T>